src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.RequestSeparator.cs

```diff
@@ -28,5 +29,19 @@ public void request_separator_at_start_of_request_is_valid()
                   .Which.UrlNode.Text.Should().Be("https://example.com");
         }
 
+        [Fact]
+        public void tokens_on_request_separator_nodes_are_parsed_into_request_separator()
+        {
+            var result = Parse(
+                """
+                ### Slow Response (Json)
+                GET https://httpbin.org/anything?page=2&pageSize=10
+                """
+                );
+
+            var requestNodes = result.SyntaxTree.RootNode.ChildNodes
```

Comment:
Curious about how this is parsed. Is a child comment node added to the request separator node?

Reply:
It's parsed as tokens into the http request separator node